src/main/java/com/hexagonal/courseregistration/course/application/domain/Duration.java

```diff
@@ -0,0 +1,14 @@
+package com.hexagonal.courseregistration.course.application.domain;
+
+import com.hexagonal.courseregistration.course.application.error.CourseException;
+import com.hexagonal.courseregistration.course.application.error.Message;
+
+public record Duration(Integer startTime, Integer endTime) {
+  public Duration {
+    if ((startTime < 1 || 9 < startTime) ||
+      (endTime < 1 || 9 < endTime) ||
+      (endTime < startTime)) {
```

Comment:
```suggestion
    if ((startTime < 1 || 9 < startTime)
      || (endTime < 1 || 9 < endTime)
      || (endTime < startTime)) {
```
자바 컨벤션은 연산자 앞에서 줄바꿈하는것으로 알고 있습니다!